docs/releases.rst

```diff
@@ -1,6 +1,22 @@
 Release Notes
 =============
 
+v0.1.5
+--------------
+
+Released September 5, 2018
+
+- set `pycryptodome` version to `3.6.6` to fix a recently discovered vulnerability
```

Comment:
Lets be precise

https://github.com/ethereum/eth-hash/blob/master/setup.py#L30

`>=3.6.6,<4`

Reply:
Also, maybe link to the issue since this is a security thing and people may care to dig in to evaluate whether to upgrade.